starknet_devnet/postman_wrapper.py

```diff
@@ -199,7 +198,6 @@ def __init__(
 
         # Create a filter to collect LogMessageToL2 events.
         w3_contract = self.mock_starknet_messaging_contract.w3_contract
-        web3_contract_create_filter_fix(w3_contract.events.LogMessageToL2)
```

Comment:
What happened here? Wasn't this filter fix important?

Reply:
This function was removed, maybe they changed something and it's not needed anymore.

I remember when I commented that during another L1L2 development tests were failing so it was needed and now without it they are passing so it's ok.